JobConfig/primary/epilog.fcl

```diff
@@ -1,7 +1,7 @@
 # use detailed physics list
 physics.producers.g4run.physics.physicsListName:ShieldingM_EMZ
-# use 'coarse' stepping in the calorimeter to save time  disble for now FIXME!
-#physics.producers.g4run.physics.minRangeRegionCuts: { CalorimeterMother : 0.1}  // {RegionName : mm }
+# use 'coarse' stepping in the calorimeter to save time
```

Comment:
This comment should be changed as it is misleading. It is production range cut for that region